### PR TITLE
Allow design options to be unset without errors on save.

### PR DESCRIPTION
**Pull request by Growthmill**
Hello!

For a site I'm building I wanted to streamline the options the site owners had, so I unhooked some of the design options for buttons. This worked, but on load they threw errors because they where not set (But viewing them still worked). So I figured I'd fix that and create a pull request.

My first formal pull request to an open source project, so please tell me if something should've been done differently.

### CODE REVIEW

File: widgets/button/button.php [hunk 1]
```diff
@@ -243,12 +243,12 @@ function get_less_variables($instance){
 		$less_vars = array(
 			'button_width' => isset( $instance['design']['width'] ) ? $instance['design']['width'] : '',
 			'has_button_width' => empty( $instance['design']['width'] ) ? 'false' : 'true',
-			'button_color' => $instance['design']['button_color'],
-			'text_color' => $instance['design']['text_color'],
+			'button_color' => isset($instance['design']['button_color']) ? $instance['design']['button_color'] : '',
+			'text_color' =>   isset($instance['design']['text_color']) ? $instance['design']['button_color'] : '',
```
Comment (braamgenis) [inline widgets/button/button.php]: This should be <pre>'text_color' =>   isset($instance['design']['text_color']) ? $instance['design']['<b>text_color</b>'] : '',</pre>

File: widgets/button/button.php [hunk 2]
```diff
@@ -320,4 +320,4 @@ function modify_instance($instance){
 	}
 }
 
-siteorigin_widget_register('sow-button', __FILE__, 'SiteOrigin_Widget_Button_Widget');
+siteorigin_widget_register('sow-button', __FILE__, 'SiteOrigin_Widget_Button_Widget');
```
Comment (braamgenis) [inline widgets/button/button.php]: Please add back the newline at the end of the file.

File: widgets/button/button.php (hunk 1 above)
Comment (Growthmill) [inline widgets/button/button.php]: Indeed. Clumpsy on my part.

**Comment by braamgenis** [comment]
Great, thanks @Growthmill !

**Pull request merged**